src/reducers/search.js

```diff
@@ -8,24 +8,23 @@ import {
   SEARCH_CLEAR,
 } from '../actions/search';
 
+let namespace;
 let loadedEntries;
 let response;
 let page;
 let searchTerm;
 
-const defaultState = Map({ isFetching: false, term: null, page: 0, entryIds: List([]), queryHits: Map({}) });
+const defaultState = Map({ isFetching: Map({}), page: 0, entryIds: List([]), queryHits: Map({}) });
 
 const entries = (state = defaultState, action) => {
   switch (action.type) {
     case SEARCH_CLEAR:
       return defaultState;
 
     case SEARCH_ENTRIES_REQUEST:
```

Comment:
Why not use an effect to dispatch a QUERY_REQUEST with namespace='search'? It may dry up this code a bit.

Reply:
For now it will stay as is, and another issue / PR should address this issue along with actions naming.

Reply:
Agreed on both counts - this definitely needs to be split between the two use cases, but that shouldn't block this PR. If someone makes an issue for that improvement it's more likely to happen.